pkg/strategy/sentinel/math.go

```diff
@@ -0,0 +1,28 @@
+package sentinel
+
+import "math"
+
+func mean(floats []float64) float64 {
+	var sum float64
+	for _, f := range floats {
+		sum += f
+	}
+	return sum / float64(len(floats))
+}
+
+func calculateMovingMeanAndStdDev(samples []float64, length int) (float64, float64) {
```

Comment:
we also have the std dev?

Reply:
I will add std dev method for floats.Slice